src/modules/sendData.js

```diff
@@ -0,0 +1,24 @@
+const nameInput = document.getElementById('name-input');
+const scoreInput = document.getElementById('score-input');
+
+const saveData = () => {
+  const URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/sN1XA4iUm9JzCAVPL1P5/scores/';
+  if (nameInput.value !== '' && scoreInput.value !== '') {
+    fetch(URL, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+      body: JSON.stringify({
+        user: nameInput.value,
+        score: scoreInput.value,
+      }),
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        console.log('Succes', data);
+      });
+  }
+};
```

Comment:
- [ ] Please make this function asyncronous. The `async` and `await` keywords enable asynchronous, promise-based behavior to be written in a cleaner style, avoiding the need to explicitly configure promise chains.
See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) to get more understanding.